src/main/webapp/scripts/searchmanagement/views/search.controller.js

```diff
@@ -413,15 +444,21 @@ angular.module('metadatamanagementApp').controller('SearchController',
         $scope.searchFilterMapping, $scope.options.sortObject.selected, false,
         $scope.searchParams.additionalSearchIndex)
         .then(function(data) {
-          var totalHitsInAdditionalIndex = 0;
+          var totalHitsInAdditionalIndex = [];
           if ($scope.searchParams.additionalSearchIndex) {
-            totalHitsInAdditionalIndex = data.responses[1].hits.total.value;
+            for (var index in $scope.searchParams.additionalSearchIndex) {
```

Comment:
Das hier finde ich irgendwie unklar. Das Array ` additionalSearchIndex` enthält ja Strings soweit ich das sehen kann und du möchtest den jeweiligen Index nutzen. Da fände ich eine reguläre For-Loop besser: `for (var index = 0; index < additionalSearchIndex.length; i++) { ... }`.